### PR TITLE
Tested with server commands

### PR DESCRIPTION
**Pull request by CerberusOne**

### CODE REVIEW

File: Client/Client/ClientTCP.h [hunk 1]
```diff
@@ -14,15 +14,19 @@
...
-#define DEFAULT_PORT	5555	
+#define DEFAULT_PORT	5000	
 #define WM_SOCKET		104
 #define PACKET_SIZE		1024
 #define MAX_THREADS		3
-#define DEFAULT_IP		"127.0.0.1"
+#define STR_MAX_SIZE	128
+#define STR_NAME		128
+
+#define DEFAULT_IP		"192.168.0.22"
```
Comment (eyyu) [inline Client/Client/ClientTCP.h:24]: Did we decide on port 5000?

Comment (eyyu) [inline Client/Client/ClientTCP.h:24]: The mutlicast IP we have will be different. TBD . just a heads up!

**Review by eyyu — COMMENTED** [review]
@CerberusOne Careful of not having new lines at the end of a file!

File: Client/Client/ClientTCP.h (hunk 1 above)
Comment (CerberusOne) [reply Client/Client/ClientTCP.h:24]: not decided, just temporary. It's what Jamie was using.

Comment (CerberusOne) [reply Client/Client/ClientTCP.h:24]: just temporary for testing

Comment (CerberusOne) [reply Client/Client/ClientTCP.h:24]: we should be able to change this with the UI once its integrated

**Review by eyyu — APPROVED** [review]

**Pull request merged**